frontend/src/components/Configuration/DroneConfiguration.jsx

```diff
@@ -16,7 +16,7 @@ import styled from '@emotion/styled';
 import { OutlinedInput } from '@mui/material';
 import Typography from '@mui/material/Typography';
 import PropTypes from 'prop-types';
-import { useMainJson } from '../../contexts/MainJsonContext';
+import { useMainJson } from '../MainJsonContext/MainJsonContext';
```

Comment:
@LaxminarayanaV7416 I would suggest to have the folder name as `contexts` as it can accommodate any other contexts in future, not only just MainJsonContext